### PR TITLE
Feature/rp/om/corrupted employee dto

### PR DESCRIPTION
**Pull request by roshnihpatel**
Created a new array list (corruptedEmpoyees ) that stores the employeeDTO that returns false when isRecordValid method is called

### CODE REVIEW

File: src/main/java/com/sparta/om/dao/EmployeeDAO.java [hunk 1]
```diff
@@ -21,8 +22,13 @@ public static ArrayList<EmployeeDTO> PopulateArray(String filename) {
             bufferedReader.readLine();
             for (String line = bufferedReader.readLine(); line != null; line = bufferedReader.readLine()) {
                 String[] records = line.split(",");
+
                 EmployeeDTO employeeDTO = new EmployeeDTO(records);
+                if(employeeDTO.isRecordValid()){
```
Comment (JamalLindo) [inline src/main/java/com/sparta/om/dao/EmployeeDAO.java]: This need to be indented and space after ") {"

File: src/main/java/com/sparta/om/dto/EmployeeDTO.java [hunk 2]
```diff
@@ -93,4 +93,9 @@ public boolean isSalaryValid() {
     public boolean isGenderValid() {
         return false;
     }
+    public boolean isRecordValid(){
+
```
Comment (omerm1) [inline src/main/java/com/sparta/om/dto/EmployeeDTO.java:97]: This line needs to be removed

File: src/main/java/com/sparta/om/dao/EmployeeDAO.java [hunk 3]
```diff
@@ -21,8 +22,13 @@ public static ArrayList<EmployeeDTO> PopulateArray(String filename) {
...
                 employees.add(employeeDTO);
+                }else{ corruptedEmployees.add(employeeDTO);
```
Comment (omerm1) [inline src/main/java/com/sparta/om/dao/EmployeeDAO.java]: "corruptedEmpl.." needs to be on a new line and "else" needs to have a space before it

File: src/main/java/com/sparta/om/dao/EmployeeDAO.java [hunk 4]
```diff
@@ -21,8 +22,13 @@ public static ArrayList<EmployeeDTO> PopulateArray(String filename) {
...
+
+                }
```
Comment (omerm1) [inline src/main/java/com/sparta/om/dao/EmployeeDAO.java]: This line needs to be removed

**Review by gursham2001 — APPROVED** [review]
This has been reviewed, it is correct

**Pull request merged**